crates/burn-dataset/src/source/huggingface/downloader.rs

```diff
@@ -292,3 +307,22 @@ fn install_python_deps(base_dir: &Path) -> Result<PathBuf, ImporterError> {
 
     Ok(venv_python_path)
 }
+
+#[cfg(test)]
+mod test {
+    #[test]
+    pub fn check_import() {
+        use crate::HuggingfaceDatasetLoader;
+        use crate::SqliteDataset;
+        use serde::Deserialize;
+
+        #[derive(Deserialize, Debug, Clone)]
+        struct MnistItemRaw {
+            pub _image_bytes: Vec<u8>,
+            pub _label: usize,
+        }
+        let _train_ds: SqliteDataset<MnistItemRaw> = HuggingfaceDatasetLoader::new("mnist")
+            .dataset("train")
+            .unwrap();
+    }
+}
```

Comment:
Good intention but I think it would slow the testing and it will break if the datasource isn't available. 

I would move to `burn-dataset/examples` folder and make executable like `speech_commands.rs` there. So that we can quickly can test it.

Reply:
Yes I was worried about this and asked on the discord server on what would've been the best course of actions without a reply.